### DISCUSSION TITLE
Customise default fields components

### DISCUSSION THREAD

**Discussion by BA7YA** [area17/twill#2442] [Q&A]
## Description

When Im trying to overwrite any default MediaField component (just want to add some new buttons), and i put it in resources/assests/js/components (I already have some overwrited components there) it's not working for me. Seems like twill is not working with my overwrited MediaField and using default one

### Steps to reproduce

1) Choose one of fields vue component in frontend/js/components
2)Copy and Add something new in this component inside your project
3) Build application though php artisan commands 

### Expected result

Changes are shown in component in Admin panel

### Actual result

Twill still using default component

### Versions

Twill version:  2.8.7

**Comment by shaifulborhan** [comment]
I need help for overriding ItemList.vue component too. Couldn't seem to call it when placed in `resources/assests/js/components `

**Comment by ifox** [comment]
I'm sorry we didn't get to you earlier but I just found out this is just a silly casing issue. Using `Mediafield.vue` and `Itemlist.vue` works fine to override them. As always, this folder is there for convenience but you can always take more control over the Twill frontend build and fully override components if you need to.
